grant-project/grant-monorepo
# Proposal style to less.

Part of #41.

### What this does
- `Proposal` styles to less.
- `ProposalUpdates` to less. (did this blind, but it should be free of errors)
- `ProposalMilestones` to less.
- `ProposalGovernance` to less.
- `ProposalComments` to less.
- `ProposalCampaignBlock` to less.

### Testing
- visit proposal details, observe that styling is correct
- make sure proposal body displays correctly and the "read more" button works
- adjust screen width and make sure responsive styles intact
- peruse bottom tabs and observe styles
- for governance, observe tab as trustee and funder, check styles for each
